Tools/cases_generator/parsing.py

```diff
@@ -138,7 +138,8 @@ class Family(Node):
 @dataclass
 class Pseudo(Node):
     name: str
-    targets: list[str]  # opcodes this can be replaced by
+    flags: list[str] | None # instr flags to set on the pseudo instruction
+    targets: list[str]      # opcodes this can be replaced by
```

Comment:
```suggestion
    flags: list[str] | None  # instr flags to set on the pseudo instruction
    targets: list[str]       # opcodes this can be replaced by
```